operator/pkg/reconciliation/defaults.go

```diff
@@ -0,0 +1,11 @@
+package reconciliation
+
+var (
+	// Provides reasonable defaults for the logger container.
+	DefaultsLoggerContainer = buildResourceRequirements(500, 256)
+
+	// Provides reasonable defaults for the configuration container.
+	DefaultsConfigInitContainer = buildResourceRequirements(1000, 256)
+
+	DefaultsReaperContainer = buildResourceRequirements(2000, 512)
```

Comment:
@jsanda what do you think is reasonable here?

Reply:
Sorry for the delayed response. I will try to have some numbers for you today.

Reply:
Should we leave the values as is and tune them later? @jsanda / @jimdickinson  what do you think?